packages/codemods/src/transforms/tanstack-query-import.ts

```diff
@@ -0,0 +1,60 @@
+import type j from 'jscodeshift'
+import { createParserFromPath } from '../utils/createParserFromPath'
+
+const IMPORT_TO_CHANGE = [
+  'useSuspenseQuery',
+  'UseSuspenseQueryResult',
+  'UseSuspenseQueryOptions',
+  'useSuspenseQueries',
+  'SuspenseQueriesResults',
+  'SuspenseQueriesOptions',
+  'useSuspenseInfiniteQuery',
+  'UseSuspenseInfiniteQueryResult',
+  'UseSuspenseInfiniteQueryOptions',
+  'usePrefetchQuery',
+  'queryOptions',
+  'infiniteQueryOption',
+]
+
+export default function transformer(file: j.FileInfo) {
+  const j = createParserFromPath(file.path)
+  const root = j(file.source)
+
+  root
+    .find(j.ImportDeclaration, {
+      source: {
+        value: (v: string) => /^@suspensive\/react-query(-\d+)?$/.test(v),
```

Comment:
I've made it so that environments where `@suspensive/react-query` is installed directly (e.g. `@suspensive/react-query-4`) can also be used.